packages/ui/src/components/va-color-input/VaColorInput.vue

```diff
@@ -111,6 +112,22 @@ const { tp } = useTranslation()
     overflow: hidden;
     position: absolute;
     pointer-events: none;
+    bottom: 0;
+  }
+
+  // Currently I could only determine the width of the color modal on Edge and Chrome
+  @supports (-ms-ime-align:auto) {
+    /* Edge specific */
+    &__hidden-input {
+      right: 14.7rem; // For Edge and Chrome the color picker modal size is 14.7rem
```

Comment:
I assume you found it with empirical method and this size is not documented somewhere. If there was source, please, share and we'll include it in source code. Thanks!